packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -188,46 +191,87 @@ const Card = React.forwardRef(function Card(
     );
 });
 
-const getComponentStyles = ({
+// Map prop values to tokens
+const styleMap = {
+    backgroundColor: {
+        "base-subtle": semanticColor.core.background.base.subtle,
+        "base-default": semanticColor.core.background.base.default,
+    },
+    borderRadius: {
+        small: border.radius.radius_080,
+        medium: border.radius.radius_120,
+    },
+    padding: {
+        none: sizing.size_0,
+        small: sizing.size_160,
+        medium: sizing.size_240,
+    },
+    elevation: {
+        none: "none",
+        low: boxShadow.low,
+    },
+} as const;
+
+// Cache for dynamically generated styles
+const dynamicStyles: Record<string, any> = {};
+
+/**
+ * Generates a unique key for caching styles based on prop combinations
```

Comment:
I added some style caching capabilities after refreshing my memory on this from other components (namely PhosphorIcon).

Reply:
The more I think, the better would be if this could be refactored as static styles (instead of dynamically generated ones). This will surely bite us very soon with the CSS modules migration and I'd strongly recommend against using this dynamic+caching approach going further (specially for new components).